README.md

```diff
@@ -96,3 +96,5 @@ SimpleCov.start
 ```
```

Comment:
<img width="622" alt="Screenshot 2022-02-07 at 10 44 14" src="https://user-images.githubusercontent.com/95872492/152773260-0c94d56a-b3e6-4154-9ff7-b52ba08fcb2f.png">
Everything looks and works as it should! Submit button works. Would have been excited to see what you could have done to make the page a bit more colourful and exciting.